graphene_sqlalchemy/converter.py

```diff
@@ -23,7 +21,7 @@ def is_column_nullable(column):
     return bool(getattr(column, "nullable", True))
 
 
-def convert_sqlalchemy_relationship(relationship, registry):
+def convert_sqlalchemy_relationship(relationship, registry, connection_field_factory):
```

Comment:
Maybe you should add a default value, instead of `default_connection_field_factory` everywhere ?

Reply:
The default it set on the main `SQLAlchemyObjectType` class. Since this function is only called by this class (via the  `construct_fields` function), I think it makes sense to keep the default logic there in one place. I don't think that `convert_sqlalchemy_relationship` nor `construct_fields` should know about the default factory.